com.dosbox.DOSBox.json

```diff
@@ -26,6 +26,17 @@
         "shared-modules/glu/glu-9.0.0.json",
         "shared-modules/SDL/SDL-1.2.15.json",
         "shared-modules/SDL/SDL_net-1.2.8.json",
+        {
+            "name": "SDL_sound",
+            "config-opts": ["--disable-static"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/sezero/SDL_sound",
```

Comment:
Hmm, I am not sure if I like using some unofficial fork repo. I think we should wait for the [download issue](https://github.com/flatpak/flatpak-builder/issues/250) to be fixed instead.

Reply:
I don't like it also, but actually that fork contains fully unmodified branches/tags from official repo, so we could rely on that for a while

Reply:
We could compare checkouts of that tag from git vs mercurial, to be sure they are identical.
@AsciiWolf would it be enough to be accepted?

Reply:
Why not using the official archive instead? You can use some mirror as noted in my comment [here](https://github.com/flatpak/flatpak-builder/issues/250#issuecomment-452115038) if the official one doesn't work for some reason.

Reply:
Still doesn't work for me, but buildbot seems hasn't the problem with official sources